### PR TITLE
PD-264 - Added index page into RoR project

### PR DESCRIPTION
**Pull request by gortonz**

### CODE REVIEW

**Comment by catherine-jones** [comment]
@gortonz you need to rebase master into your branch and get my fix for the build.

File: app/helpers/dashboard_helper.rb [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+            else
+              'aus-width-one-half'
+          end
+    return cls
+  end
+
+  def DashboardHelper.truncateDashboardName(name)
+
+  end
+
```
Comment (catherine-jones) [inline app/helpers/dashboard_helper.rb]: @gortonz is the helper below being used for anything? I think we could probably remove it

File: app/helpers/dashboard_helper.rb [hunk 2]
```diff
@@ -1,2 +1,34 @@
 module DashboardHelper
+
+  def DashboardHelper.sizeToStyle(size)
+    cls = case size
+            when 'extra-small' then
+              'aus-width-one-fourth'
```
Comment (tobyhede) [inline app/helpers/dashboard_helper.rb]: @catherine-jones Is it possible to actually make the CSS more semantic here? So have classes for the charts that set the width based on more human names than 'aus-width-one-fourth'

Comment (tobyhede) [inline app/helpers/dashboard_helper.rb]: cc @elisechant

**Comment by gortonz** [comment]
A pull request before Toby adding more back-end stuffs

File: app/helpers/dashboard_helper.rb (hunk 2 above)
Comment (catherine-jones) [inline app/helpers/dashboard_helper.rb]: @tobyhede yeah we could change them that's not hard but not sure what would be more 'human' than what they are already. I was just following what 18F have done.  Suggestions maybe sm-col, thirds?

Comment (tobyhede) [inline app/helpers/dashboard_helper.rb]: For charts, couldn't we just use small, medium etc?

**Comment by catherine-jones** [comment]
@goton LGTM

**Pull request merged**